src/service/crypto-impl.js

```diff
@@ -215,6 +215,26 @@ export class Crypto {
       data
     ));
   }
+
+  /**
+   * Returns a Uint8Array of the given length full of cryptographically
+   * random values if win.crypto is available.
+   * If win.crypto, is not available, returns null.
+   * @param {number} length The length of the string to return
+   * @return {Uint8Array|null}
+   */
+  getSecureRandomBytes(length) {
```

Comment:
This actually doesn't need to live here, we can include it directly in the double click code. This crypto service was made so that we could seamlessly polyfill, without the dev needing to be aware of the code path. Since we're not using the polyfill here, we can do it directly in the code.

This'll mean 0 size increase for the runtime!

Reply:
Done